### PR TITLE
feat: initial logflare cli integration

### PR DESCRIPTION
**Pull request by Ziinc**
Initial Logflare integration for allowing users to access their logs + charts in supabase studio during local development

Development notes:
- use `INTERNAL_IMAGE_REGISTRY=docker.io go run . start` for pointing to public docker
- to quickly test if locally built docker image works, in logflare repo, run `docker-compose build` and then run above command to use the locally tagged image.

### CODE REVIEW

File: internal/utils/docker.go [hunk 1]
```diff
@@ -180,7 +180,7 @@ func GetRegistryAuth() string {
 }
 
 // Defaults to Supabase public ECR for faster image pull
-const defaultRegistry = "public.ecr.aws"
+const defaultRegistry = "docker.io"
```
Comment (sweatybridge) [inline internal/utils/docker.go:183]: revert this when images are mirrored https://github.com/Logflare/logflare/pull/1418

File: internal/utils/templates/initial_schemas/14.sql [hunk 2]
```diff
@@ -25,6 +25,16 @@ CREATE SCHEMA IF NOT EXISTS _realtime;
...
 
+--
+-- Name: _analytics; Type: SCHEMA; Schema: -; Owner: postgres
+--
+
+CREATE SCHEMA IF NOT EXISTS _analytics;
+
+
+ALTER SCHEMA _analytics OWNER TO postgres;
+ALTER USER postgres WITH REPLICATION;
```
Comment (sweatybridge) [inline internal/utils/templates/initial_schemas/14.sql:36]: The `postgres` user already has replication attribute by default

```bash
postgres=# \du postgres
                                                     List of roles
 Role name |                         Attributes                         |                  Member of
-----------+------------------------------------------------------------+----------------------------------------------
 postgres  | Superuser, Create role, Create DB, Replication, Bypass RLS | {supabase_auth_admin,supabase_storage_admin}
```

File: internal/start/start.go [hunk 3]
```diff
@@ -95,6 +96,7 @@ func run(p utils.Program, ctx context.Context, fsys afero.Fs, excludedContainers
 	for _, name := range excludedContainers {
 		excluded[name] = true
 	}
+	enableLogs := len(utils.Config.Analytics.GcpProjectId) > 0 && len(utils.Config.Analytics.GcpProjectNumber) > 0
```
Comment (sweatybridge) [inline internal/start/start.go:99]: Might be nice to add an enabled flag to toml directly. for eg.

```toml
[analytics]
enabled = true
gcp_project_id = ""
```

File: internal/utils/templates/initial_schemas/14.sql (hunk 2 above)
Comment (Ziinc) [reply internal/utils/templates/initial_schemas/14.sql:36]: 👍

**Review by sweatybridge — APPROVED** [review]

**Pull request merged**